web/packages/design/src/Button/buttons.story.tsx

```diff
@@ -85,20 +85,23 @@ export const Buttons = () => {
             </Fragment>
           ))}
         </tbody>
-      </Table>{' '}
```

Comment:
Because of this `{' '}`, Prettier would automatically remove any empty lines between tags within this component.